<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Focus</class>
 <widget class="QDialog" name="X2Focus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>326</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>326</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Oasis focuser settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>99999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>240</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>20</y>
        <width>184</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Focuser temperature source :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Air</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tube</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mirror</string>
       </property>
      </item>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>48</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>On</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>48</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>48</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Fans :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>120</y>
        <width>144</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Air Temperature :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>140</y>
        <width>144</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tuber Temperature :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>160</y>
        <width>144</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mirrorr Temperature :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="airTemp">
      <property name="geometry">
       <rect>
        <x>184</x>
        <y>120</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>22.5 ºC</string>
      </property>
     </widget>
     <widget class="QLabel" name="tubeTemp">
      <property name="geometry">
       <rect>
        <x>184</x>
        <y>140</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>22.5 ºC</string>
      </property>
     </widget>
     <widget class="QLabel" name="mirrorTemp">
      <property name="geometry">
       <rect>
        <x>184</x>
        <y>160</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>22.5 ºC</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>200</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Restore position on connect</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>70</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
